uast/ann/ann.go

```diff
@@ -53,14 +53,20 @@ type Rule struct {
 	rules      []*Rule
 }
 
+const (
```

Comment:
Yes, I was refering to the `head` constant, which I thought was public. You're absolutely right.